### ISSUE TITLE
Design Implementation 

### ISSUE THREAD

**Issue by allthesignals** [MAPC/dyee-dashboard#16]
There are some early mockups - collecting them here. 

![image](https://cloud.githubusercontent.com/assets/5004319/24427670/d1f3184c-13d9-11e7-923d-ad8894d81c82.png)
![image](https://cloud.githubusercontent.com/assets/5004319/24427672/d6d4c9f0-13d9-11e7-9e5a-ee53c5a7778f.png)
![image](https://cloud.githubusercontent.com/assets/5004319/24427693/e668ba16-13d9-11e7-992e-718843dc7dd2.png)
![image](https://cloud.githubusercontent.com/assets/5004319/24427714/fcd86828-13d9-11e7-955d-1024c36507e5.png)

**Comment by allthesignals** [comment]
@arouault is this a system font? I can't find it on google fonts. I don't think it's a normal cross-platform system font. http://web.mit.edu/jmorzins/www/fonts.html.

**Comment by allthesignals** [comment]
Test

**Comment by allthesignals** [comment]
Just going to use https://fonts.google.com/specimen/Source+Sans+Pro instead

**Comment by arouault** [comment]
It's a typekit font. Here are the details:

embed code place near <head> :

<script src="https://use.typekit.net/rsf1xqd.js"></script>
<script>try{Typekit.load({ async: true });}catch(e){}</script>

If you’re using a plugin or service that asks for a Typekit Kit ID, use this: rsf1xqd

For CSS selectors:
use the Selectors section to apply fonts to any HTML tag, class, or id. You can also add the default class to your markup. Then, click Publish to see those changes on your site.

.tk-franklin-gothic-urwdelete 

current domain for kit:  dyee-dashboard.pagefrontapp.com/

**Comment by allthesignals** [comment]
@arouault hi, the domain we're using is youthmatch.mapc.org - the pagefrontapp works, but the font won't load when people go to youthmatch.mapc.org.

**Comment by arouault** [comment]
Is that because the domain is wrong in typekit? @allthesignals

**Comment by allthesignals** [comment]
@arouault yes- you used dyee-dashboard.pagefrontapp.com but when people go to youthmatch.mapc.org, Adobe will see the origin as youthmatch.mapc.org and reject it.

**Comment by allthesignals** [comment]
@arouault I need to know the full font name as it's used in Typekit - it's not working.

**Comment by allthesignals** [comment]
I also need you to add 'localhost' so I can test this... is there a shared MAPC Typekit account we can use so I can get what I need?
